aten/src/ATen/native/Blas.cpp

```diff
@@ -152,6 +164,16 @@ Tensor vdot(const Tensor &self, const Tensor &other){
     return at::dot(self, other);
   }
 
+  if (self.is_conj()) {
+    if (other.is_conj()) {
+      return at::vdot(self.conj(), other.clone());
```

Comment:
This doesn't look right, you are still calling `vdot`, but you are conjugating the first arg. `vdot(a,b)` cannot be equal to `vdot(a_conjugated, b)`? (`.clone` on `other` shouldn't matter, it resolves conj bit, but it doesn't change logical interpretation of data in `other`, whereas `conj` on `self` changes logical interpretation of data). 
Do we have tests that would check cross product of dot/vdot with conj bit set or not set on either argument?

Reply:
vdot(conj(A), conj(B)) = sum_i(a_i * conj(b_i)). So in fact you can compute this as vdot(B, A)

Reply:
yeah totally. I have local changes to update it to that. just haven't pushed them yet